test/unit/lifecycle/defined.js

```diff
@@ -0,0 +1,9 @@
+import { define } from '../../../src/index';
+
+describe('lifecycle/defined', () => {
+  it('should add the [defined] attribute when the element is upgraded', () => {
```

Comment:
Good point. I'll add a test and push it to master once the release happens for 1.1.